### PR TITLE
[xxx] A few snag fixes

### PR DESCRIPTION
**Pull request by darokel**
### Context

Fixes some of the issues spotted in the snag.

### CODE REVIEW

File: app/views/publish/courses/accredited_provider/new.html.erb [hunk 1]
```diff
@@ -23,24 +23,26 @@
           <% if @provider.accredited_bodies.length > 0 %>
             <%= render partial: "provider_suggestion", collection: @provider.accredited_bodies.sort_by { |k| k[:provider_name] }, locals: { form: fields } %>
 
-            <div class="govuk-radios__divider">or</div>
+            <% unless FeatureService.enabled?(:accredited_provider_search) %>
```
Comment (avinhurry) [inline app/views/publish/courses/accredited_provider/new.html.erb:26]: Makes sense, so we will remove the additional field when we remove the feature flag? Is it worth leaving a comment so we remember to remove it?

File: config/settings.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ support_email: becomingateacher@digital.education.gov.uk
 campaign_email: itt.engagement@education.gov.uk
 
 search_ui:
-  base_url: http://localhost:3002
+  base_url: https://find.localhost
```
Comment (avinhurry) [inline config/settings.yml:8]: What's this?

Comment (defong) [reply config/settings.yml:8]: Can we get rid of it?

File: app/views/publish/courses/accredited_provider/new.html.erb (hunk 1 above)
Comment (darokel) [reply app/views/publish/courses/accredited_provider/new.html.erb:26]: Yup, you just left one on this PR 😂

File: config/settings.yml (hunk 2 above)
Comment (darokel) [reply config/settings.yml:8]: The `view on find link` in dev didn't load the correct url

File: app/views/publish/courses/accredited_provider/new.html.erb (hunk 1 above)
Comment (avinhurry) [reply app/views/publish/courses/accredited_provider/new.html.erb:26]: lol I meant in the code?

File: config/settings.yml (hunk 2 above)
Comment (darokel) [reply config/settings.yml:8]: It's still used

File: app/views/publish/courses/accredited_provider/new.html.erb (hunk 1 above)
Comment (darokel) [reply app/views/publish/courses/accredited_provider/new.html.erb:27]: Screen Recording (Safari)

I also left it in the code 😜

**Review by avinhurry — APPROVED** [review]

**Pull request merged**